incubator/gitea/requirements.yaml

```diff
@@ -0,0 +1,5 @@
+dependencies:
+  - name: postgresql
+    version: 0.8.1
```

Comment:
You can use a SemVer range for the version. You can read about the details at https://github.com/Masterminds/semver#basic-comparisons

Reply:
Hey @mattfarina I think you're wrong. A lot of the stable charts use the exact same dependency with `0.8.1`

Reply:
The latest version of that chart is `0.8.9` which means there have been 8 patch releases you'll not get. These include bug fixes and minor improvements.

In the [chart best practices guide](https://docs.helm.sh/chart_best_practices/#requirements-files), in the helm docs, it recommends using ranges.

There are a number of charts here that aren't doing that. We likely need to make fixes to those other charts to clean up quality. PRs are welcome.